src/test/java/chatbot/ParserTest.java

```diff
@@ -0,0 +1,148 @@
+package chatbot;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import chatbot.exceptions.DeleteMissingFieldException;
+import chatbot.exceptions.EventMissingFieldException;
+import chatbot.exceptions.InvalidTaskIndexException;
+import chatbot.exceptions.MarkMissingFieldException;
+import chatbot.exceptions.MissingFieldException;
+import chatbot.tasks.EventTask;
+import chatbot.tasks.Task;
+
+public class ParserTest {
```

Comment:
Consider renaming your test methods to be more descriptive, the recommended format:
`featureUnderTest_testScenario_expectedBehavior`

Reply:
I see, thanks!